data.txt

```diff
@@ -1 +1 @@
-My=Foo
\ No newline at end of file
+My=Data
```

Comment:
I wasn't expecting this file to be committed.

Reply:
Yeah, since, the other PR wasn't merged with the "no data.txt" and "auto creation of data.txt" when the server is started up, it is still a part of Master. I didn't want to do the changes twice, and I figured I would fix it during the merge. Is that a good plan or should I have added the changes?

Reply:
That sounds reasonable to me.